cert/load.go

```diff
@@ -14,8 +14,11 @@ import (
 	"path/filepath"
 	"sort"
 	"strings"
+
+	"github.com/hashicorp/go-multierror"
```

Comment:
Unless one of the other maintainers has a counterpoint (@nathanejohnson or @aaronhurt ?), I would prefer we use the `errors` package for handling error concatenation.

The code can be updated to something like this:

```go
var errs []error

....
errs = append(errs, err)
...

return certs, errors.Join(errs...)
```

I'll defer to others if they prefer we use go-multierror, however.

Reply:
Note that `errors.Join` will return `nil` if there are no non-nil values in the provided slice, so you do not have to test for nil explicitly before appending to the `errs` slice.

Reply:
if we do use the errors package I'll update this and [bgp/bgp_nonwindows.go](https://github.com/fabiolb/fabio/blob/5d113395f3d08045ebad183245fd69d9576dd317/bgp/bgp_nonwindows.go#L231) too.

Reply:
I'm generally in favor of fewer deps whenever possible, even when they're as stable as hashi's go-multierror.

Reply:
Trying the same manual test of deleting some keys, (using errors package instead) the error response is as follows:

```
2025/01/23 23:29:28 [ERROR] cert: Failed to load certificates. cert: cannot load certificate consul.service.consul-cert.pem
cert: invalid certificate fabio.service.consul-key.pem. tls: failed to find any PEM data in key input
cert: cannot load certificate nomad.service.consul-key.pem
cert: invalid certificate fabio.service.consul-cert.pem. tls: failed to find any PEM data in key input
```